packages/compass-aggregations/src/modules/update-view.js

```diff
@@ -94,6 +94,7 @@ export const updateView = () => {
         const metadata = {
           namespace: viewNamespace,
           isReadonly: true,
+          preferencesReadOnly: state.preferencesReadOnly,
```

Comment:
Where is this being used/why is this being added here?

Reply:
Removed.